CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+## v0.10.0 [2023-10-03]
+
+_Breaking changes_
+
+- Removed `instances` column from `terraform_resource` table. ([#64](https://github.com/turbot/steampipe-plugin-terraform/pull/64))
+- Updated tables having columns `arguments` and `lifecycle` with JSON data to return `nil` if empty instead of `{}`. ([#64](https://github.com/turbot/steampipe-plugin-terraform/pull/64))
```

Comment:
```suggestion
- All `arguments` and `lifecycle` columns now return `null` instead of `{}` if empty. ([#64](https://github.com/turbot/steampipe-plugin-terraform/pull/64))
```